examples/light-switch-app/cc13x4_26x4/src/AppTask.cpp

```diff
@@ -151,6 +151,10 @@ TimerHandle_t sOTAInitTimer = 0;
 // be set to false.
 bool isAppStarting = true;
 
+#if (CHIP_CONFIG_ENABLE_ICD_LIT && CHIP_CONFIG_ENABLE_ICD_DSLS)
+static bool sitModeRequested;
+#endif // CHIP_CONFIG_ENABLE_ICD_LIT && CHIP_CONFIG_ENABLE_ICD_DSLS
```

Comment:
Documentation seems to say that lit must be enabled first. Are we able to add an assert() somewhere that this is the case?

Thinking

```cpp
#if CHIP_CONFIG_ENABLE_ICD_DSL
#if CHIP_CONFIG_ENABLE_ICD_LIT
static bool sitModeRequested;
#else
#error "DSL requires LIT as a prerequisite"
#endif
#endif
```

Reply:
Or maybe we can do this in GN somewhere (that seems to maybe be better as it would apply to all build targets)

Reply:
This assert exists in src/app/icd/server/BUILD.gn.